### ISSUE TITLE
69-1-1 View | Zoom is not needed

### ISSUE THREAD

**Issue by mlos100** [ChordPro/chordpro#487]
**Severity**: Minor
**Category**: GUI

The menu item, View | Zoom appears to toggle maximizing the application window. Windows already provides this functionality by double-clicking the application window title bar and via the Maximize icon in the upper right corner of the window.  I am fairly certain very similar functionality is available by Linux and MacOS.

Rationale: One less functionality to maintain

FWIW: Entering Shift-Ctrl-Z on Windows did not seem to do anything. The window did not maximize when it wasn't nor did it become unmaximized when I maximized the window using View | Zoom or the Maximize icon.

chordpro filename: 69-1-1
About: 6.069_001
Windows 10

**Comment by sciurius** [comment]
@Desbeers Can you comment? IIRC there was something on macOS with zoom and fullscreen...

**Comment by Desbeers** [comment]
@sciurius

Yes, we discussed this before; first this was a ‘faux’ full-screen (on macOS at least). You made it into ‘Zoom’ that I consider handy because on macOS the ‘green’ button does *not* zoom by default unless you hold the ‘option’ key.

The keyboard shortcuts works fine on Linux and macOS.

**Comment by sciurius** [comment]
On my Fedora system with MATE and OpenBox, double clicking the title bar has no effect.

**Comment by mlos100** [comment]
Would you consider (apparently) reverting to naming the menu item "Full-Screen" (instead of zoom)?  FWIW: It seems to be a true full-screen on Windows -- that is, clicking the Maximize icon or double-clicking title bar returns the window size to what it was before selecting View | Zoom.

**Comment by sciurius** [comment]
There's Zoom (100 - 150% - ...)
There's Maximize (maximum size but with title/menu bars)
There's Full Screen (maximum size without title/menu bars)

macOS may differ.

Until we have sorted out how to handle these on all platforms I'd rather leave it as it is now.
